src/firebase/user/sign_up_user_response.cr

```diff
@@ -0,0 +1,26 @@
+module Google
+  class FirebaseAuth
+    class SignUpUserResponse
+      include JSON::Serializable
+
+      getter kind : String?
+
+      @[JSON::Field(key: "idToken")]
+      getter id_token : String?
+
+      @[JSON::Field(key: "displayName")]
+      getter display_name : String?
+
+      getter email : String?
+
+      @[JSON::Field(key: "refreshToken")]
+      getter refresh_token : String?
+
+      @[JSON::Field(key: "expiresIn", converter: Google::StringTypeConverter(Int64))]
+      getter expires_in : Int64?
```

Comment:
Is this an epoch or span? It would be worth converting to a `Time` type either way (unless that's the pattern across the rest of this library)

Reply:
span of seconds